pkg/export/otel/common.go

```diff
@@ -87,9 +87,6 @@ func getResourceAttrs(hostID string, service *svc.ID) []attribute.KeyValue {
 	for k, v := range service.Metadata {
 		attrs = append(attrs, k.OTEL().String(v))
 	}
-
-	attrs = append(attrs, otelResourceAttribs...)
```

Comment:
@marctc Is there a way to actually keep this here instead? Because `getResourceAttrs` lies both on the metrics and traces codepath, the intention was to have a single point in which we decorate both traces and metrics with the contents of `OTEL_RESOURCE_ATTRIBUTES`.

Reply:
Metrics still uses the metrics SDK, so the env var will be populated automatically

Reply:
Discussed in a different places. For some reason the values are not populated so we need to explicitly pass the env var.